site/content/en/latest/tasks/traffic/kubernetes-traffic-distribution-support.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Kubernetes Traffic Distribution Support"
```

Comment:
should we call this Zone Aware Routing instead ?

Reply:
I'm good either way but was trying to think about it from the perspective a user would be approaching this. In the context of kubernetes they'd look for "traffic distribution" but for Envoy terms yes Zone Aware Routing makes sense.
Let me know which you prefer please!

Reply:
prefer Zone Aware Routing, thanks